From 2d30977c75ae518a5f333508af7f79bbd5137081 Mon Sep 17 00:00:00 2001
From: Kaleb S KEITHLEY <kkeithle@redhat.com>
Date: Tue, 12 Apr 2016 07:11:07 -0400
Subject: [PATCH 67/80] packaging: gluster-ganesha requires pacemaker, etc., on RHEL6

HA packages in RHEL6 are missing dependencies. (Fedora and RHEL7
have good dependencies.)

Backport of
> master
> Change-Id: I4e80b569ea89f97e8a110aee2f4554175aa0ec8b
> BUG: 1322801
> http://review.gluster.org/13870
> release-3.7
> Change-Id: I411eeb5c4ee1a2f1ec0869ca4a9bbacd8c4a5fcb
> BUG: 1322808
> http://review.gluster.org/13871

Change-Id: Iee4b6716f9781c1886a4012136b141514f68fe97
BUG: 1292034
Signed-off-by: Kaleb S KEITHLEY <kkeithle@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/71967
Reviewed-by: Soumya Koduri <skoduri@redhat.com>
Reviewed-by: Jiffin Thottan <jthottan@redhat.com>
---
 glusterfs.spec.in |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index 7366bcd..89037aa 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -395,6 +395,9 @@ Group:            Applications/File
 
 Requires:         %{name}-server%{?_isa} = %{version}-%{release}
 Requires:         nfs-ganesha-gluster, pcs, dbus
+%if ( 0%{?rhel} && 0%{?rhel} == 6 )
+Requires:         cman, pacemaker, corosync
+%endif
 
 %description ganesha
 GlusterFS is a distributed file-system capable of scaling to several
@@ -1934,6 +1937,9 @@ end
 
 
 %changelog
+* Tue Apr 12 2016 Kaleb S. KEITHLEY <kkeithle@redhat.com>
+- pacemaker dependencies on RHEL6 (#1292034)
+
 * Tue Mar 8 2016 Kaleb S. KEITHLEY <kkeithle@redhat.com>
 - %%pre, %%post etc. scriptlet cleanup, ... -p /sbin/ldconfig (#1315935)
 
-- 
1.7.1

